### DISCUSSION TITLE
Anyway to disable automatically resizing a specific app window on start?

### DISCUSSION THREAD

**Discussion by hemna** [nikitabobko/AeroSpace#689] [questions-and-answers]
Hello,
  I'm developing an ios app on my mac with Aerospace as my window manager on my mac.  the problem I have with Aerospace is that when I launch my app, it gets resized to a huge size to fit the rest of the desktop due to tiling.  I would like to disable the tiling for this specific app/window only.

I have tried to force the layout to floating for that window only, but unfortunately it changes the layout for the entire desktop and the apps.

```
[[on-window-detected]]
if.app-id = 'NULL-APP-BUNDLE-ID'
run = 'layout floating'
```

Is there a command I can place in that run to disable tiling for that specific app-id?  I need to keep the same size of that window when I launch/quit/edit code/relaunch development cycle.

Thanks!

**Comment by gldtn** [comment]
Just of the top of my head, you probably need to figure out the app id, maybe with `aerospace list-apps`

```
[[on-window-detected]]
if.app-id = 'SET-APP-ID-HERE'
run = 'layout floating'
```

example:
```
[[on-window-detected]]
if.app-id = 'com.apple.finder'
run = 'layout floating'
```

See: 
https://nikitabobko.github.io/AeroSpace/commands#list-apps
https://nikitabobko.github.io/AeroSpace/guide#on-window-detected-callback

**Comment by hemna** [comment]
The app id was correct in my post.  It's an app I'm developing and the id isn't bundled yet into a formal package.

I got that app id from running aerospace list-apps.

But setting 'layout floating' will change the entire layout to floating.  I just want that 1 window to be floating.

**Comment by gldtn** [comment]
That should work, I just tested here on my end and it only floats the set apple id. Does the other apps have the same/similar id? Did you try using `if.app-name-regex-substring` or `if.window-title-regex-substring`?

**Comment by hemna** [comment]
The app id is null, because it's under development.
